src/views/FarmAuction/components/AuctionCakeBurn.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react'
+import CountUp from 'react-countup'
+import { Text, Flex, Skeleton } from '@pancakeswap/uikit'
+import { useFarmAuctionContract } from 'hooks/useContract'
+import { useTranslation } from 'contexts/Localization'
+import { usePriceCakeBusd } from 'state/farms/hooks'
+import { getBalanceNumber } from 'utils/formatBalance'
+import { ethersToBigNumber } from 'utils/bigNumber'
+
+const AuctionCakeBurn: React.FC = () => {
+  const [burnedCakeAmount, setBurnedCakeAmount] = useState(null)
+  const { t } = useTranslation()
+  const farmAuctionContract = useFarmAuctionContract()
+  const cakePriceBusd = usePriceCakeBusd()
+
+  const burnedAmountAsUSD = cakePriceBusd.times(burnedCakeAmount).toNumber()
+
+  useEffect(() => {
+    const fetchBurnedCakeAmount = async () => {
```

Comment:
Not sure how far down the page this appears but could be worth using the intersectionObserver? 🗑️ this suggestion if it's near the top of the page

Reply:
You need to scroll to it, but it fetches just one thing once and is super fast, don't think it worth the hassle